src/components/Books.js

```diff
@@ -1,20 +1,60 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../redux/books/booksSlice';
+
+const Books = () => {
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+
+  const dispatch = useDispatch();
+
+  const handleAddBook = () => {
+    dispatch(
+      addBook({
+        item_id: uuidv4(),
+        title,
+        author,
+        category: 'Unknown',
+      }),
+    );
+
+    setTitle('');
+    setAuthor('');
+  };
 
-function Books() {
   return (
     <div className="book-container">
       <h1>ADD A NEW BOOK</h1>
       <form>
-        <input type="text" className="add-book" placeholder="Book title" />
-        <select name="category" id="book">
+        <input
+          type="text"
+          className="add-book"
+          placeholder="Book title"
+          value={title}
+          onChange={(e) => setTitle(e.target.value)}
+        />
+        <select
+          name="category"
+          id="book"
+          value={author}
+          onChange={(e) => setAuthor(e.target.value)}
+        >
```

Comment:
- Good Job Implementing the project requirements. However, there are still some issues that you need to work on. When I want to add a book, it is not being added. This may be caused by your form that's not well implemented. Kindly make sure to change this and add add The Author placeholder as well as Adding the select option for categories separately, instead of doing it all together, to avoid conflicts when submitting the form. 
- Kindly make sure the user is able to add a book according to [this requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/use_redux_in_react_components_v2.md#:~:text=Add%20a%20%3CButton,and%20author))

Reply:
@GedeonTS Done with this change